Orange/tests/test_preprocess.py

```diff
@@ -83,6 +84,15 @@ def test_remove_nan_classes_multiclass(self):
         self.assertEqual(len(table), 1)
 
 
+class TestScaling(unittest.TestCase):
+    def test_scaling_x(self):
+        table = Table("iris")
+        dist1 = distribution.get_distribution(table, table.domain.attributes[0])
+        self.assertTrue(Scaling.mean(dist1) > 5)
```

Comment:
We prefer to use self.assertGreater, since the error it prouduces is more informative. Also, could you include the checks for median, stdev and span as well (they could be done in two pairs, mean&stdev and median&span)?

This should increase the coverage enough to make the codecov happy.